influxql/parser.go

```diff
@@ -311,8 +313,8 @@ func (p *Parser) parseCreateRetentionPolicyStatement() (*CreateRetentionPolicySt
 	// Parse optional DEFAULT token.
 	if tok, pos, lit = p.scanIgnoreWhitespace(); tok == DEFAULT {
 		stmt.Default = true
-	} else {
-		p.unscan()
+	} else if tok != EOF {
+		return nil, newParseError(tokstr(tok, lit), []string{"DEFAULT"}, pos)
```

Comment:
I discussed `FORCE` with @pauldix and he's OK with it as long as it's not too difficult. My concern is that without `FORCE` users may cause themselves real grief it they mistakenly drop the wrong server, and it's the only owner of data.

While one could argue that if the data is that important, no server should be the only owner of the data. However during certain cluster recovery situations, one could imagine times when replication factors may drop below the required levels for a short while (multiple nodes being swapped out to deal with operational issues).